.github/workflows/ci.yml

```diff
@@ -45,13 +73,32 @@ jobs:
       - name: Cache Rust dependencies
         uses: Swatinem/rust-cache@v2
 
+      - name: Run tests with rust_native_crypto
+        shell: pwsh
+        if: runner.os == 'Windows'
+        env:
+          RUST_BACKTRACE: "1"
+          FEATURES: ${{needs.get-features.outputs.rust-native-features}}
+        run: |
+          cargo test --no-default-features --features "$env:FEATURES"
```

Comment:
Would you please try to achieve code coverage for the rust-native-crypto path as well?